AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage.py

```diff
@@ -5,154 +5,134 @@
-
-EXPECTED_LINES = ["Activator Script: Activated", "Deactivator Script: Deactivated"]
-controller_dict = {
-    "name": "Controller",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "controller.scriptcanvas"),
-}
-activated_dict = {
-    "name": "ActivationTest",
-    "status": "inactive",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "activator.scriptcanvas"),
-}
-deactivated_dict = {
-    "name": "DeactivationTest",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "deactivator.scriptcanvas"),
-}
-ENTITY_TO_ACTIVATE_PATH = "Configuration|Properties|Variables|EntityToActivate|Datum|Datum|value|EntityToActivate"
-ENTITY_TO_DEACTIVATE_PATH = "Configuration|Properties|Variables|EntityToDeactivate|Datum|Datum|value|EntityToDeactivate"
-WAIT_ONE_SECOND = 1.0
 # fmt: off
 class Tests():
-    controller_exists    = ("Successfully found controller entity",  "Failed to find controller entity")
-    activated_exists     = ("Successfully found activated entity",   "Failed to find activated entity")
-    deactivated_exists   = ("Successfully found deactivated entity", "Failed to find deactivated entity")
-    start_states_correct = ("Start states set up successfully",      "Start states set up incorrectly")
     game_mode_entered    = ("Successfully entered game mode",         "Failed to enter game mode")
     lines_found          = ("Successfully found expected prints",    "Failed to find expected prints")
     game_mode_exited     = ("Successfully exited game mode", "Failed to exit game mode")
 # fmt: on
 
-class ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage():
-    """
-    Summary:
-     Verify that the On Entity Activated/On Entity Deactivated nodes are working as expected
+class testEntity:
 
-    Expected Behavior:
-     Upon entering game mode, the Controller entity will wait 1 second and then activate the ActivationTest
-     entity. The script attached to ActivationTest will print out a message on activation. The Controller
-     will also deactivate the DeactivationTest entity, which should print a message.
+    def __init__(self, name: str, status: str, file: str):
 
-    Test Steps:
-     1) Create temp level
-     2) Create all the entities we need for the test
-     3) Validate the entities were created and configured
-     4) Start the Tracer
-     5) Enter Game Mode
-     6) Wait one second for graph timers then exit game mode
-     7) Validate Print message
+        import os
+        import azlmbr.paths as paths
 
-    Note:
-     - This test file must be called from the Open 3D Engine Editor command terminal
-     - Any passed and failed tests are written to the Editor.log file.
-            Parsing the file or running a log_monitor are required to observe the test results.
+        self.name = name
+        self.status = status
```

Comment:
entity_state is probably better here than status

Reply:
agreed. updated variable name